_dev/apps/ui/.storybook/mock/merchant-center-account.ts

```diff
@@ -5,10 +5,10 @@ export const merchantCenterAccountNotConnected: MerchantCenterAccountContext = {
   ...state.googleMerchantAccount,
 };
 
-export const merchantCenterAccountConnected: MerchantCenterAccountContext = {
+export const merchantCenterAccountUnverifiedAndUnclaimed: MerchantCenterAccountContext = {
```

Comment:
Je vois dans le code que `merchantCenterAccountConnected` est utilisé dans beaucoup d'autre fichier de storybook, il faut pas modifier le nom dans ces fichiers aussi (banners-ads-overviews.stories.ts) ?

Reply:
The diff makes you believe I renamed this variable, but you'll find it a bit lower in this file (l.24). Its content remains the same.